R/class-plotdata-bar.R

```diff
@@ -2,6 +2,7 @@ newBarPD <- function(.dt = data.table::data.table(),
                          variables = veupathUtils::VariableMetadataList(),
                          value = character(),
                          barmode = character(),
+                         overlayValues = character(),
```

Comment:
Bit confused - shouldn't `overlayValues` be passed to the `newPlotdata` a few lines below? Ah yes, I see it is in, for example, boxplot. Just an oversight!  Or a test! :smile: